### DISCUSSION TITLE
How can I set new configuration values for b-form-datepicker (BFormDatepicker, BCalendar properties) in browser?

### DISCUSSION THREAD

**Discussion by vanphong1310** [bootstrap-vue/bootstrap-vue#5791] [Q&A]
### Describe the bug

I need globally change all label text of b-form-datepicker, I follow the intruction in document [https://bootstrap-vue.org/docs/reference/settings#setting-new-configuration-values](url) but it don't work.

### Steps to reproduce the bug

Call Vue.use after load bootstrap-vue.js by script tag.
```
Vue.use(BootstrapVue, {
    BFormDatepicker: {
        ...
    },
});
```

### Expected behavior

All label text are change.

### Versions

**Libraries:**

- BootstrapVue: 2.10.1
- Bootstrap: 4.3.1
- Vue: 2.6.12

**Comment by jacobmllr95** [comment]
@vanphong1310 Can you please update to the latest BootstrapVue version (v2.17.1) and see if the issues still exist.

Here is an example with the settings properly configured:
https://codesandbox.io/s/gallant-https-hz1nr?file=/index.js

**Comment by vanphong1310** [comment]
@jackmu95 Thank you for your help! I was updated to v2.17.0 but there is no change. I use bootstrap-vue in browser by loading it with script tag, so I can't import library like your example :(
I found this code in dist/bootstrap-vue.js line 1467:
```
  var vueUse = function vueUse(VuePlugin) {
    /* istanbul ignore next */
    if (hasWindowSupport && window.Vue) {
      window.Vue.use(VuePlugin);
    }
    /* istanbul ignore next */


    if (hasWindowSupport && VuePlugin.NAME) {
      window[VuePlugin.NAME] = VuePlugin;
    }
  };
````
It call Vue.use(BootstrapVue), so I can't call Vue.use again with my options :(
